hal/src/lib.rs

```diff
@@ -98,6 +101,10 @@ pub mod timer_traits;
 #[cfg(all(feature = "unproven", feature = "dma"))]
 pub mod dmac;
 
+// The SAMD11 hardware supports USB too, but the SAMD11 HAL currently doesn't
+#[cfg(all(feature = "usb", not(feature = "samd21"), not(feature = "min-samd51g")))]
+compile_error!("The 'usb' feature is enabled, but not a specific chip that supports USB");
```

Comment:
Doesnt this do the wrong thing (crash) when 
 `device` and `usb` features are enabled?

Reply:
This would not compile with only `usb` and `device`, but I doubt there is any device-independent usb-specific code...

Reply:
I think we'll need to modify the condition to consider `library`:
```Rust
#[cfg(all(feature = "usb", not(feature = "samd21"), not(feature = "min-samd51g"), not(feature = "library")))]
```